aspnet/client-side/bundling-and-minification.rst

```diff
@@ -1,8 +1,315 @@
-.. include:: /../common/stub-topic.txt
+Bundling and Minification
+=========================
+
+By `Erik Reitan`_ 
+
+Bundling and minification are two techniques you can use in ASP.NET 5 to improve page load performance for your web application. Bundling makes it easy to combine or bundle multiple files into a single file. Minification performs a variety of different code optimizations to scripts and CSS, which results in smaller file sizes.
+
+This article explains and shows how to use the bundling and minification features included with the ASP.NET 5 Web Application template in `Visual Studio 2015 <http://go.microsoft.com/fwlink/?LinkId=517106>`_. 
+
+Overview
+--------
+Because bundling combines multiple files into a single file, it reduces the number of requests to the server that is required to retrieve and display a web asset, such as a web page. You can create CSS, JavaScript and other bundles. Fewer files, means fewer HTTP requests from your browser to the server or from the service providing your application. This results in improved first page load performance.
+
```

Comment:
Prefer my more concrete intro and graph. Current browser allow more than 6, but youcan update that.